### DISCUSSION TITLE
play.chakra-ui / codesandbox

### DISCUSSION THREAD

**Discussion by JimmyJag** [chakra-ui/chakra-ui#9362] [General]
I noticed that https://play.chakra-ui.com/ returns an error because it is still using extendTheme from v2. I saw a previous discussion about play.chakra-ui possibly being broken but it appears to predate v3. 

I couldn't find a codesandbox for v3 so I created one which you can find here: https://codesandbox.io/p/sandbox/chakra-v3-6nn5vc
Just in case anyone else is searching for a quick way to share code snippets. 

Edit: Didn't intend this to be a question but I'll leave it open in case play being down has slipped the radar of the incredible Chakra team during the transition to V3

**Comment by BBezzola** [comment]
On that regard, I created a PR to lock the dependencies for the playground for V2 [here](https://github.com/chakra-ui/play.chakra-ui.com/pull/12)
I might make another one for this playground so that users are able to switch from V2 to V3 theming and play around

**Comment by segunadebayo** [comment]
play.chakra-ui.com was the v2 playground.

Use a v3 StackBlitz or CodeSandbox from the docs instead.
